doc/development/deprecations.rst

```diff
@@ -9,6 +9,13 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+* ``MultiControlledX`` is the only controlled operation that still supports specifying control
+  values with a bit string, it is also the only controlled operation that still takes a
+  ``control_wires`` argument to its constructor. Both behaviours will be deprecated.
```

Comment:
Thanks! Ok let's proceed with this. Though note that `ControlledQubitUnitary` also has a `controlled_wires` argument.

Reply:
We can probably put `ControlledQubitUnitary` in the next deprecation cycle. I'm including `control_wires` in this deprecation mainly because it is already half deprecated.